docs/website/docs/plus/features/pipeline-runner.md

```diff
@@ -0,0 +1,189 @@
+---
+title: "Dlt+ Runner"
+description: Run pipelines with the Dlt+ Runner
+keywords: ["runner", "pipeline", "retry", "trace"]
+---
+# Runner
+
+
+The dlt+ Runner provides a production-ready run command for your pipelines. It offers robust error handling, retry mechanisms, and 
+near atomic trace storage to destinations of your choice.
+The runner can be configured for each pipeline via your `dlt.yml`, or you can use it directly
+in your python code.
+
+## Key features
+
+- **Trace storage** to a configurable destination for debugging and monitoring pipeline executions
+- **Automatic retry policies** with configurable backoff strategies
+- **Clean state management** to ensure consistent pipeline runs
+
+## Configuration
+
+The Runner is configured through the `run_config` section in your `dlt.yml` file.
+
+### Complete configuration example
+
+```yaml
+pipelines:
+  my_pipeline:
+    source: my_source
+    destination: duckdb
+    run_config:
+      run_from_clean_folder: true
+      store_trace_info: true
+      retry_policy:
+        type: backoff
+        max_attempts: 5
+        multiplier: 2
+        min: 1
+        max: 10
+      retry_pipeline_steps: ["load"]
+```
+
+## Run from clean folder
+
+The `run_from_clean_folder` parameter controls whether the pipeline starts from a clean state before running.
```

Comment:
The parameter name could be confusing for the people who don't have a deep understanding of how dlt works. Do you think we can still change the name? For example, `fresh_run` or `force_resync` or `sync_from_destination`?

Reply:
Hmm, i guess we could, as the runner is released pretty silently for now and probably nobody is using it. But I am not convinced that your suggestions are better.. lets chat